playbooks/group_vars/prometheus/public.yml

```diff
@@ -161,7 +163,17 @@ prometheus_rules:
       annotations:
         summary: "MongoDB instance [[ $labels.node ]] ([[ $labels.instance ]]) down."
         description: "[[ $labels.node ]] ([[ $labels.instance ]]) of job [[ $labels.job ]] is down."
-
+  - name: "HueyRules"
```

Comment:
@shimulch, @0x29a there should be nothing specific to Huey (Rename all instances Huey to say Redis queue or Redis list) or Ocim here and the queue key name should not be hardcoded and instead be specified using a variable.

Reply:
We can either go with `redis` queued tasks or a very specific `Ocim queue size` alert so that the FFs/ops reviewer don't need extra context to know what's going on.

Reply:
Changed to ``OcimQueueSizeRule``. I guess now ``OCIM_QUEUE_REDIS_KEY`` & ``PROMETHEUS_ALERT_OCIM_QUEUE_SIZE_THRESHOLD`` have to move to ``secrets`` repository.